### PR TITLE
MEPTS-1349 | TXCURR CHANGES DECEMBER

### PR DESCRIPTION
**Pull request by helderjosue**

### CODE REVIEW

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/TxCurrCohortQueries.java [hunk 1]
```diff
@@ -216,31 +196,31 @@ public CohortDefinition getTxCurrCompositionCohort(String cohortName, boolean cu
...
             EptsReportUtils.map(
-                getTransferredOutOnProgramEnrollment(),
-                "onOrBefore=${onOrBefore},location=${location}"));
+                getPatientsWhoStartedArtAfterDecember2023AndHasDrugPickupByReportEndDate(),
+                "endDate=${onOrBefore},location=${location}"));
 
     String compositionString;
     if (currentSpec) {
       compositionString =
-          "(1 OR 2 OR 3 OR 4 OR 5) AND NOT ((6 OR 7 OR 8 OR 9 OR 11 OR ((10 OR 15 OR transferredOutProgram) AND mostRecentSchedule)) AND NOT 12) AND NOT (13 OR 14)";
```
Comment (psmatsinhe) [inline api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/TxCurrCohortQueries.java:243]: You not using the 5, you should remove from the map

**Review by psmatsinhe — CHANGES_REQUESTED** [review]
Have a look

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/TxCurrCohortQueries.java (hunk 1 above)
Comment (helderjosue) [reply api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/TxCurrCohortQueries.java:243]: done

**Review by psmatsinhe — APPROVED** [review]
Looks good to me

**Pull request merged**